src/routes/api/index.js

```diff
@@ -1,18 +1,8 @@
-const router = require("express").Router();
+import { Router } from 'express';
+import authRoute from './socialRoute';
```

Comment:
The import name should be the same as the file name. Here should be import socialRoute from '/socialRoute'